aspnetcore/blazor/state-management.md

```diff
@@ -226,21 +537,31 @@ During prerendering:
 * An interactive connection to the user's browser doesn't exist.
 * The browser doesn't yet have a page in which it can run JavaScript code.
 
-`localStorage` or `sessionStorage` aren't available during prerendering. If the component attempts to interact with storage, an error is generated similar to:
-
-> JavaScript interop calls cannot be issued at this time. This is because the component is being prerendered.
+`localStorage` or `sessionStorage` aren't available during prerendering. If the component attempts to interact with storage, an error is generated explaining that JavaScript interop calls cannot be issued because the component is being prerendered.
 
 One way to resolve the error is to disable prerendering. This is usually the best choice if the app makes heavy use of browser-based storage. Prerendering adds complexity and doesn't benefit the app because the app can't prerender any useful content until `localStorage` or `sessionStorage` are available.
 
-To disable prerendering, open the `Pages/_Host.cshtml` file and change the `render-mode` of the [Component Tag Helper](xref:mvc/views/tag-helpers/builtin-th/component-tag-helper) to <xref:Microsoft.AspNetCore.Mvc.Rendering.RenderMode.Server>.
+To disable prerendering, open the `Pages/_Host.cshtml` file and change the `render-mode` of the [Component Tag Helper](xref:mvc/views/tag-helpers/builtin-th/component-tag-helper) to <xref:Microsoft.AspNetCore.Mvc.Rendering.RenderMode.Server>:
+
+```cshtml
+<component type="typeof(App)" render-mode="Server" />
+```
 
-Prerendering might be useful for other pages that don't use `localStorage` or `sessionStorage`. To keep prerendering enabled, defer the loading operation until the browser is connected to the circuit. The following is an example for storing a counter value:
+Prerendering might be useful for other pages that don't use `localStorage` or `sessionStorage`. To retain prerendering, defer the loading operation until the browser is connected to the circuit. The following is an example for storing a counter value:
 
 ```razor
 @using Microsoft.AspNetCore.ProtectedBrowserStorage
```

Comment:
Any occurrences of `Microsoft.AspNetCore.ProtectedBrowserStorage` should be changed to `Microsoft.AspNetCore.Components.Web.Extensions` 😃

Reply:
There are two sections here tho ... there's the <5.0 section that retains the references from the experimental package and the new >=5.0 section with the new namespace. I'll double check it.

Reply:
Yeah ... this content is in the <5.0 area of the topic. It's correct here. We'll keep this until 3.1 falls out of support.